config/default.yml

```diff
@@ -148,6 +148,71 @@ Style/BarePercentLiterals:
     - percent_q
     - bare_percent
 
+Style/Blocks:
```

Comment:
Is there a graceful way to rename the cop and still pick up configuration from the old name for backwards-compatibility with people's existing configuration?